include/Core/Posts/Modules/DeletePostsByCommentsModule.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Posts\Modules;
+
+use BulkWP\BulkDelete\Core\Posts\PostsModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Delete Posts by Comments Module.
+ *
+ * @since 6.1.0
+ */
+class DeletePostsByCommentsModule extends PostsModule {
+	/**
+	 * Base parameters setup.
+	 */
+	protected function initialize() {
+		$this->item_type     = 'posts';
+		$this->field_slug    = 'comments';
+		$this->meta_box_slug = 'bd_by_comments';
+		$this->action        = 'delete_posts_by_comments';
+		$this->cron_hook     = 'do-bulk-delete-comments';
+		$this->scheduler_url = 'http://bulkwp.com/addons/scheduler-for-deleting-posts-by-comments/?utm_source=wpadmin&utm_campaign=BulkDelete&utm_medium=buynow&utm_content=bd-st';
+		$this->messages      = array(
+			'box_label'  => __( 'By Post Comments', 'bulk-delete' ),
+			'scheduled'  => __( 'The selected posts are scheduled for deletion', 'bulk-delete' ),
+			'cron_label' => __( 'Delete Post By Comments', 'bulk-delete' ),
+		);
+	}
+
+	/**
+	 * Render Delete posts by comments box.
+	 */
+	public function render() {
+		?>
+		<h4><?php _e( 'Delete Posts based on the number of comments', 'bulk-delete' ); ?></h4>
+
+		<!-- Comments start-->
+		<fieldset class="options">
+			<table class="optiontable">
+				<tr>
+					<td scope="row" colspan="2">
+						<?php _e( 'Delete posts that have comments', 'bulk-delete' ); ?>
+					</td>
+					<td>
+						<select name="smbd_<?php echo esc_attr( $this->field_slug ); ?>_count_op" id="smbd_<?php echo esc_attr( $this->field_slug ); ?>_count_op">
```

Comment:
We have a helper function called `render_number_comparison_operators` in `Renderer` class. Let's use that here.

Reply:
Option value differs(Please refer slack for more detailed info).  Kindly advice.

Reply:
@esudalaimadan 

Let's change the helper function to use the values that you are using.

After making that change, find the place where the helper function is currently being used and then change the values in that module as well.

I did a quick search and we are using the helper function in only one other place.